app/views/accounts/show.html.slim

```diff
@@ -55,10 +55,7 @@ h1.hide = t('titles.account')
         action: @view_model.personal_key_action_partial
 
   - if TwoFactorAuthentication::BackupCodePolicy.new(current_user).visible?
-      = render 'account_item',
-        name: t('forms.backup_code.title'),
-        content: content_tag(:em, @view_model.backup_codes_content),
-        action: @view_model.backup_codes_partial
+      = render 'backup_codes'
```

Comment:
I should probably also convert this view to HTML/ERB

Reply:
So going forward, if any changes we do for a PR include changes to a slim file, we should convert it to ERB right away?